### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by Developers-codz**
Refactor Class variables for card images and avatar
Separated label and input 
Worked on modals
Did some minor fixes

Thank you for reviewing my PR

### CODE REVIEW

**Comment by Vishnu-Aithal** [comment]
Looks Good !
some small findings -


image

-Something is blocking bottom part of aside section when you scroll

-navbar can add align items center to vertically center "Beads UI" logo

**Comment by Developers-codz** [comment]
> 

Done!

File: css/style/button.css [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
 }
 .btn-primary {
   color: var(--text-light);
   background-color: var(--primary-color);
   font-size: 1.2rem;
-  border: none;
   padding: 0.5rem 1rem;
 }
 .btn-secondary {
   padding: 0.5rem 1rem;
```
Comment (b0llu) [inline css/style/button.css:13]: can use px if the value is less than rem

**Review by arjun-computer-geek — APPROVED** [review]
Looks good to me

**Review by b0llu — APPROVED** [review]
Looks good. Suggested one change everything else looks ok

File: css/style/button.css (hunk 1 above)
Comment (Developers-codz) [reply css/style/button.css:13]: Is it compulsory to use px if value is less in rem?

**Review by Vishnu-Aithal — APPROVED** [review]

**Pull request merged**